scripts/common/_common.sh

```diff
@@ -30,5 +30,12 @@ export HOST_DIR=$OUTPUT_ROOT/corehost
 [ -z "$CONFIGURATION" ] && export CONFIGURATION=Debug
 [ -z "$NOCACHE" ] && export NOCACHE=""
 
+[ -z "$BRANCH_TYPE"] && export BRANCH_TYPE=dev
```

Comment:
I hadn't gone so far as implementing that interface yet. My expectation was that build.sh would accept a cmd line parameter, and use it to set this environment variable.

Note also that windows changes still need to be made. My intention here is to keep the ball rolling forward as we agree to things.